app/views/layouts/homepage.html.erb

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="<%= t("sufia.document_language", default: '') %>">
+<head>
+  <%= render "layouts/head_tag_content"%>
+</head>
+
+<body>
+<%= render '/masthead' %>
+
+<div class="image-masthead">
+  <div class="background-container" style="background-image: url('<%= Site.instance.banner_image? ? Site.instance.banner_image.url : Sufia.config.banner_image %>')"></div>
```

Comment:
Ah, I see the difference now. Maybe this would be well-suited to be a helper in sufia that we can override locally? That'd at least spare us the pain of overriding this whole layout..